litestar/testing/client/base.py

```diff
@@ -54,6 +54,7 @@ def fake_asgi_connection(app: ASGIApp, cookies: dict[str, str]) -> ASGIConnectio
         "http_version": "1.1",
         "extensions": {"http.response.template": {}},
         "app": app,  # type: ignore[typeddict-item]
```

Comment:
so we need to keep the "app" key here ?

Reply:
Yeah, since we don't want to break a bunch of stuff that may rely on it. We're explicitly documenting this usage in our docs.